lib/data_migrate/tasks/data_migrate_tasks.rb

```diff
@@ -4,8 +4,12 @@ module DataMigrateTasks
       extend self
       def migrations_paths
         @migrations_paths ||= begin
-          if Rails.application && Rails.application.paths["data/migrate"]
+          if DataMigrate.config.data_migrations_path
+            DataMigrate.config.data_migrations_path
+          elsif Rails.application && Rails.application.paths["data/migrate"]
```

Comment:
oh, interesting .. so there are two ways to do this? what if we only go with `Rails.application.paths["data/migrate"]`. What's the downside?

Reply:
Do you mean remove the Rails.application check? @ilyakatz

Reply:
i mean, this seems to indicate that there are two ways configure data migrate. One way is to configure `DataMigrate.config` and the way is to configure `Rails.application.paths["data/migrate"]`. Other places only use `DataMigrate.config` so I'm asking if we really need two ways to configure it? Maybe we should only stick with one?

Reply:
I agree, we should keep it uniform